src/main/java/org/opensearch/performanceanalyzer/PerformanceAnalyzerPlugin.java

```diff
@@ -269,6 +269,7 @@ public PerformanceAnalyzerPlugin(final Settings settings, final java.nio.file.Pa
                         eventLogFileHandler,
                         MetricsConfiguration.SAMPLING_INTERVAL,
                         QUEUE_PURGE_INTERVAL_MS,
+                        0,
```

Comment:
Why are we passing the `lastCleanupTimeBucket` as an argument to `EventLogQueueProcessor` ?

Reply:
Updated the check in the cleanup method